src/AssetsPage/index.jsx

```diff
@@ -11,6 +15,10 @@ class AssetsPage extends React.Component {
     };
   }
 
+  componentDidMount() {
+    this.props.getAssets('course-v1:edX+DemoX+Demo_Course');
```

Comment:
We should start thinking about how we're going to get the course id into React/Redux land. Presumably we'll pass it in from Studio when we instantiate the `AssetsPage` component?

Reply:
That seems like a reasonable assumption to make at this stage.